src/templates/loadTestWorkflow.yaml

```diff
@@ -0,0 +1,89 @@
+name: Perform Load Test
+on: workflow_dispatch
+jobs:
+  runLoad:
+    name: Run K6 Load
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          sparse-checkout: |
+            script.js
+            convert.js
+            k6.deb
+            reports/
+      - name: Verify vars are set and fail if not
+        run: |
+          if [ -z "${{vars.VUS}}" ]; then
+            echo "Please set VUS in workflow variables"
+            exit 1
+          fi
+          if [ -z "${{vars.DURATION}}" ]; then
+            echo "Please set DURATION in workflow variables"
+            exit 1
+          fi
+          if [ -z "${{vars.MESH_ENDPOINT}}" ]; then
+            echo "Please set MESH_ENDPOINT in workflow variables"
+            exit 1
+          fi
+      - name: Install ca-certificates
+        run: sudo apt-get install -y ca-certificates
+      - name: Install gnupg2
+        run: sudo apt-get install -y gnupg2
+      - name: Install K6
+        run: sudo apt-get install ./k6.deb
```

Comment:
Working on wget or curl based download instead of using a local version.

The k6 package from ubuntu repo wont work according to their docs. We have to download their arm64 deb file and install it manually, at least till they fix it.

Reply:
Done